CONTRIBUTING.md

```diff
@@ -1,4 +1,25 @@
-## Contributing
+# Contributing
+___
+## 🛠 ObjC to Swift Migration
+Do you want to help with the ObjC to Swift migration? Awesome! We have a [project tracker](https://github.com/RevenueCat/purchases-ios/projects/3), some [milestones](https://github.com/RevenueCat/purchases-ios/milestones), lots of [issues tagged](https://github.com/RevenueCat/purchases-ios/labels/swift%20migration) for the migration. Here's how you can get going.
+
+### You can do this!
+We've tagged a number of issues as [you can do this!](https://github.com/RevenueCat/purchases-ios/labels/you%20can%20do%20this%21). These will generally not require much (or any!) working knowledge of our SDK, they are tasks like migrating a single `enum` into Swift, or a uncomplicated `model` from objc to swift.
```

Comment:
In wonder if the `enum`s might actually end up being some of the trickiest ones, given that if they're public, moving them into Swift will functionally make them `internal` until we finally use a single module

Reply:
Yeah, I've found some enums that were easy, and others that were very annoying, but not hard.